cmd/litcli/accounts.go

```diff
@@ -243,11 +240,8 @@ func updateAccount(ctx *cli.Context) error {
 var listAccountsCommand = cli.Command{
 	Name:      "list",
 	ShortName: "l",
-	Usage:     "Lists all off-chain accounts.",
-	Description: `
-	Returns all accounts that are currently stored in the account
-	database.
-	`,
+	Usage:     "List all off-chain accounts.",
+	Description: "Returns all accounts that are currently stored in the account	database.",
```

Comment:
Nit: There's some spaces after the word "account" here which are shown when running: `litcli accounts c --help`

Reply:
This was a lingering tab character. It has been replaced by a single space.